x/qgb/abci.go

```diff
@@ -2,23 +2,32 @@ package qgb
 
 import (
 	"errors"
+	"fmt"
 
 	sdkerrors "cosmossdk.io/errors"
+
 	"github.com/celestiaorg/celestia-app/x/qgb/keeper"
 	"github.com/celestiaorg/celestia-app/x/qgb/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// SignificantPowerDifferenceThreshold the threshold of change in the validator set power
-// that would need the creation of a new valset request.
-const SignificantPowerDifferenceThreshold = 0.05
+const (
+	// SignificantPowerDifferenceThreshold the threshold of change in the validator set power
+	// that would need the creation of a new valset request.
+	SignificantPowerDifferenceThreshold = 0.05
+
+	// AttestationPruningThreshold the minimum number of recent attestations that will always be present
+	// in state.
+	AttestationPruningThreshold = 100
```

Comment:
That was the original plan. But we decided to also keep a certain number of attestations if the last unbonding height is so recent as a safety measure.

For example, if at block height 100, we have a new unbonding height, then, at height 101, we will delete all attestations before height 100, leaving only the last valset in state. 
If the orchestrators are running and up to date, then, that would be no issue. However, if they have some high latency, or... they might miss signing an attestation or two before they're pruned.

This won't be problematic in case it only happened for 1 or 2 orchestrators. However, if it happens for 1/3, that means the QGB will halt.

So, we decided to always keep a minimum number of attestations in state to account for this kinda failures on the orchestrators side.

Note: This won't be an issue if orchestrator is connected to an archive node. But we can't make that assumption for all orchestrators

Reply:
[nit] if we are retaining this constant, proposal to rename to something like

```suggestion
	// MinimumAvailableAttestations is the minimum number of attestations that will always be present
	// in state. Older attestations will be pruned.
	MinimumAvailableAttestations = 100
```

because it's not clear what happens below or above the "threshold". In other words are attestations pruned below the threshold or above the threshold?